lib/src/components/_stacks-tables.less

```diff
@@ -46,21 +46,39 @@
         padding: @su8;
         border-top: 1px solid @brc-lighter;
         text-align: left;
-        vertical-align: top;
+        vertical-align: middle;
+        border-right: 1px solid @brc-lighter;
+        &:last-child {
+            border-right: none;
+        }
+    }
+    th {
+        font-weight: normal;
+        color: @fc-dark;
```

Comment:
If you set the font color to dark here, there's no need to restate the color below in `thead th`.